ecpp

```diff
@@ -112,7 +138,95 @@ def main():
     except KeyboardInterrupt:
         print(n)
         raise
-    print("Number is prime")
+    print("\n")
+    print("[+] number is prime\n")
+
+def correlate(moduli, certs=None, verify_opt=None, primo_opt=None):    
+    missing_cert_flag = False 
+    cert_files = []
+    if certs is None:
+        certs = certs_dir
+    for path, subdirs, files in os.walk(certs):
+        for name in files:
+            cert_files.append(os.path.join(path, name))
+    cert_for_moduli  = []
+    no_cert_for_moduli = []
+    status = None
+    cert_dict = {}
+    for cert in cert_files:
+        if cert.endswith('.out'):
+            path = cert
+            cert_candidate = candidate_finder(path)
+            cert_dict[cert_candidate] = path
```

Comment:
this whole collection of certs and numbers in them probably would be a good thing to put in a separate function